riff-raff/app/resources/PrismLookup.scala

```diff
@@ -28,8 +28,11 @@ class PrismLookup(config: Config, wsClient: WSClient, secretProvider: SecretProv
     val apiCredentials = data.keys flatMap {
       case key@KeyPattern(service) =>
         data.datum(key, app, stage, stack).flatMap { data =>
-          secretProvider.lookup(service, data.value).map { secret =>
-            service -> ApiCredentials(service, data.value, secret, data.comment)
+          if(service == "aws-roles") {
+            Some(service -> ApiCredentials(service, data.value, " ", data.comment))
+          } else {
+            secretProvider.lookup(service, data.value).map { secret =>
+              service -> ApiCredentials(service, data.value, secret, data.comment)}
```

Comment:
If we have both a role and a static, both are added to the keychain, right? But later in AWS.provider we don't try one and then fallback to the other? I reckon we should either do that fallback trick or single out which one we will use here. Both don't need to be in the keychain if we won't use them both.

Not quite sure how to do that.